packages/collabswarm/src/collabswarm-document.ts

```diff
@@ -71,7 +71,7 @@ export class CollabswarmDocument<
   ChangeFnType,
   PrivateKey,
   PublicKey,
-  DocumentKey,
```

Comment:
Weird, this undoes a change by my autosave Prettier, in theory both using the same `.prettierrc` with `"trailingComma": "all",`